app/views/exam_templates/_form.html.erb

```diff
@@ -0,0 +1,20 @@
+<%= form_for exam_template,
+             url: assignment_exam_templates_path,
+             html: {multipart: true} do |f| %>
```

Comment:
For hashes the conventional style is the include a blank space after the `{` and before the `}